src/components/Foremost/Foremost.tsx

```diff
@@ -52,6 +52,10 @@ const ForemostTool = () => {
         },
     });
 
+    const handleAdvancedModeToggle = () => {
+        form.updateValues({ checkedAdvanced: !form.values.checkedAdvanced });
```

Comment:
Hi @YashKanodia,

Please review the below exception for the return type.

To fix this you can:

- [ ] Check the interface and ensure that it exists.